test/patterns/string_concat/test_concat_string.py

```diff
@@ -63,7 +63,7 @@ def test_string_with_quotes(self):
         self.assertEqual(lines, [
             352, 372, 373, 379, 380, 408, 409, 418, 422, 429, 430, 438, 456, 466,
             479, 494, 499, 509, 562, 563, 604, 605, 610, 701, 995, 1009, 1012, 1029,
-            1032, 1080, 1258, 1267, 1337, 1386, 1414
+            1032, 1080, 1188, 1258, 1267, 1337, 1366, 1372, 1386, 1414
```

Comment:
Can you explain why does test changed?

Reply:
it  was not found earlier:
`return "\"" + (String) p.getDefault() + "\".to_string()";`
`example = "\"" + ((param.example != null) ? param.example : "") + "\".to_string()";`
It's correct. Seems previous code didn't find double quotes